interface/forms/newpatient/common.php

```diff
@@ -321,384 +338,371 @@ function cancelClickedOld() {
-                                        $perow = sqlQuery("SELECT count(*) AS count FROM issue_encounter WHERE " .
-                                            "pid = ? AND encounter = ? AND list_id = ?", array($pid, $encounter, $list_id));
-                                        if ($perow['count']) {
-                                            echo " selected";
-                                        }
-                                    } else {
-                                        // For new encounters the invoker may pass an issue ID.
-                                        if (!empty($_REQUEST['issue']) && $_REQUEST['issue'] == $list_id) {
-                                            echo " selected";
-                                        }
-                                    }
-                                    echo ">" . text($tcode) . ": " . text($irow['begdate']) . " " .
-                                        text(substr($irow['title'], 0, 40)) . "</option>\n";
-                                }
-                                ?>
-                            </select>
-                            <p><i><?php echo xlt('To link this encounter/consult to an existing issue, click the '
-                                        . 'desired issue above to highlight it and then click [Save]. '
-                                        . 'Hold down [Ctrl] button to select multiple issues.'); ?></i></p>
-                        </div>
-                    </div>
-                </fieldset>
-                <?php
-            }
-            ?>
+                            </fieldset>
+                            <?php
+                        }
+                    ?>
+                </div>
+            </div>
             <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
             <div class="form-row">
-                <div class="col-sm-12 text-left position-override">
+                <div class="col-sm-12 text-left position-override pl-3">
                     <div class="btn-group" role="group">
                         <button type="button" class="btn btn-primary btn-save" onclick="top.restoreSession(); saveClicked(undefined);"><?php echo xlt('Save'); ?></button>
                         <?php if ($viewmode || empty($_GET["autoloaded"])) { // not creating new encounter ?>
-                            <button type="button" class="btn btn-secondary btn-cancel" onClick="return cancelClickedOld()"><?php echo xlt('Cancel'); ?></button>
+                            <button type="button" class="btn btn-cancel" onClick="return cancelClickedOld()"><?php echo xlt('Cancel'); ?></button>
                         <?php } else { // not $viewmode ?>
-                            <button class="btn btn-secondary btn-cancel link_submit" onClick="return cancelClickedNew()"><?php echo xlt('Cancel'); ?></button>
```

Comment:
I'm assuming there was no javascript tied to the link_submit class here?

Reply:
Resolved